lib/elixir/lib/system.ex

```diff
@@ -369,13 +369,43 @@ defmodule System do
   """
   @spec get_env(binary) :: binary | nil
   def get_env(varname) when is_binary(varname) do
+    case fetch_env(varname) do
+      :error -> nil
```

Comment:
Maybe we should make it `get_env(varname, default // nil)` to follow other `get` interfaces?